Resources/doc/reference/introduction.rst

```diff
@@ -1,5 +1,15 @@
 Introduction
 ============
 
-This small introduction will try to explain the basics concepts behind the
-``ArticleBundle``.
+The first goal of SonataArticle Bundle is to provide a starter kit to manage contents.
+
+An ``Article`` will be like a skeleton content who will contain the main logic of what you want to build, but not the content itself. For example, this is where you can configure content types, or publication status, titles, categories...
+A ``Fragment`` will handle a part of a the content. It can be very simple like only a title. Or more complex, with an associated media, a rich text editor, or something else.
+
+When we build this bundle we kept in mind the PageBundle logic, which we thought was really interesting. If you understand the logic between ``Pages`` and ``Blocks``, you will understand a lot of similarity between ``Articles`` ands ``Fragments``.
+
+.. note::
+
+    We decided to create a separate bundle from ``SonataNewsBundle`` because we consider a news matches to blog need, with comments and notification.
```

Comment:
I don't understand this sentence. Can you explain the idea you want to convey here?